packages/trader/src/Stores/base-store.js

```diff
@@ -464,15 +465,13 @@ export default class BaseStore {
         if (typeof this.preSwitchAccountDisposer === 'function') {
             this.preSwitchAccountDisposer();
         }
-        this.pre_switch_account_listener = null;
     }
 
     @action.bound
     disposeSwitchAccount() {
         if (typeof this.switchAccountDisposer === 'function') {
             this.switchAccountDisposer();
         }
-        this.switch_account_listener = null;
```

Comment:
Why are we removing this?

Reply:
If this is to prevent `switch_account_listener` from being called when it is `null`.  Can we check if the function is `null` before being called?

Reply:
Also, you should change these in `base-store` of core package too

Reply:
Basically changing the value of the listener to `null` is wrong. It's not needed. It's just set once and we can't initialize it later manually.
Checking for null is not useful as well per the fact that we can't change its value on switching phase.
Also I didn't find any refs to `base-store` of core. I think it's a duplicate file and should be removed.
@oskar-binary Am I right?

Reply:
Ah I find the refs :)) OK Will update in `core` too.

Reply:
Yes, I agree. But when we want to check if a store is subscribed to `onSwitchAccount`, we can find it by checking if `switch_account_listener` is set or not and after disposing, it should not have reference to the callback function of the `child-store` IMO.

`cashier-store` still has ref to these function in `base-store`

Reply:
Yea. I changed it and added a `null check` for listener sent from child classes. It worked.